FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -313,20 +322,74 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     </support>
                                 </supportDesc>
                             <layoutDesc>
-                                <layout columns="2" writtenLines="21" corresp="#p1_i1"></layout>
-                                <layout columns="2" writtenLines="35" corresp="#p1_i2"></layout>
-                                <layout columns="2" writtenLines="30" corresp="#p1_i3"></layout>
+                                <layout columns="2" writtenLines="21" corresp="#p1_i1">
+                                    <locus from="2ra" to="2vb"></locus>
+                                    <locus from="7ra" to="113vb"></locus>   
+                                    <locus from="128rb" to="128vb"></locus>
+                                    <note corresp="#textarea1">Data on text area dimensions are taken from <locus target="#1r #7r #10"/>.</note>
```

Comment:
you do not need `corresp="textarea1"` here, as we are inside the `layout `element